tools/template_executor.go

```diff
@@ -0,0 +1,44 @@
+package tools
```

Comment:
I think this could be in the web package or a sub package of web, or is it used somewhere else?

Reply:
In `web` and in `tools.Middlewares` should I still move it, I would be okay with both actually as it is much more used in web than in tools 👍

Reply:
You're right :+1: